feat: update recipe intro and some metadata (#20)
infinitered/ignite-cookbook: docs/intro.md

---
sidebar_position: 1
title: Intro to Recipes
description: Welcome to the Ignite Cookbook! This is a collection of recipes for common patterns in Ignite projects.
tags:
  - Intro
last_update:
  author: Dan Edwards
---

# Welcome to the Ignite Cookbook!

> This is a collection of recipes for common patterns that don't quite belong in the Ignite boilerplate. We'll be adding to this over time, so be sure to check back often!

Reviewer: ```suggestion
This is a collection of recipes for common patterns that we use sometimes at Infinite Red or in the Ignite community but don't quite belong in the Ignite boilerplate directly. We'll be adding to this over time, so be sure to check back often!
```

## What is a Recipe?

A recipe is a collection of steps that you can follow to accomplish a common task or pattern in your Ignite project. Recipes are written to be as simple as possible, and are meant to be a starting point for you to build upon. You can use them as-is, or you can use them as a reference to help you create your own solutions.

## Who is this for?

> This is for anyone who wants to learn more about Ignite, and how to use it to build their own apps. You don't need to be an expert in React Native or Ignite to follow along. If you're new to Ignite, we recommend you start with the [new docs page to explore the boilerplate](https://github.com/infinitered/ignite/tree/master/docs/).

Reviewer: ```suggestion
This is for anyone who wants to extend their Ignite-based React Native app with additional functionality, and how to use it to build their own apps. You don't need to be an expert in React Native or Ignite to follow along. If you're new to Ignite, we recommend you start with the [new docs page to explore the boilerplate](https://github.com/infinitered/ignite/tree/master/docs/). You can also use these patterns if you're not using Ignite, as in the end Ignite is just React Native!
```

> ## How can I trust these recipes?

Reviewer: ```suggestion
## Are these recipes tested and up-to-date?
```

> All recipes are written by the Ignite team, and are reviewed by the community. If you find a recipe that is out of date, or doesn't work, please [open an issue](https://github.com/infinitered/ignite-cookbook/issues). It's our goal to keep these recipes up to date and working, so we appreciate your help!

Reviewer: ```suggestion
All recipes are written or reviewed by the Ignite team, and are also reviewed by the community. If you find a recipe that is out of date, or doesn't work, please [open an issue](https://github.com/infinitered/ignite-cookbook/issues). It's our goal to keep these recipes up to date and working, so we appreciate your help!
```
